lib/_emerge/actions.py

```diff
@@ -1052,8 +1046,8 @@ def _calc_depclean(settings, trees, ldpath_mtimes, myopts, action, args_set, spi
             if not pkgs_for_cp or pkg not in pkgs_for_cp:
                 raise AssertionError(
                     "package expected in matches: "
-                    + "cp = %s, cpv = %s matches = %s"
-                    % (pkg.cp, pkg.cpv, [str(x) for x in pkgs_for_cp])
+                    f"cp = {pkg.cp}, cpv = {pkg.cpv} "
```

Comment:
wonder if we want a comma

Reply:
After the `cpv = ...` bit, so that it looks like `cp =, cpv =, matches =`?

Reply:
That's it, yes please! (Sorry for being vague)